src/IntegratorConfig.php

```diff
@@ -296,6 +301,14 @@ public function getManifestsDirectory(): string
         return $this->getProjectRootDirectory() . static::MANIFESTS_DIRECTORY;
     }
 
+    /**
+     * @return string
+     */
+    public function getGlossaryFilePath(): string
```

Comment:
As integrator will be part of sdk, I don't think that it will be possibility to configure it.

Reply:
But we might have to internally

Reply:
I see just one reason - testing purpose\internally but I don't think that we have any other options.

Reply:
Still fine to keep for now imo.
This can change in the future, this path actually must change, as current path is an anti pattern